hs_app_timeseries/tests/test_timeseries_metadata.py

```diff
@@ -66,133 +73,502 @@ def tearDown(self):
+        self.assertEquals(self.resTimeSeries.metadata.variable.variable_code, 'USU36')
+        self.assertEquals(self.resTimeSeries.metadata.variable.variable_name, 'Temperature')
+        self.assertEquals(self.resTimeSeries.metadata.variable.variable_type, 'Water Quality')
+        self.assertEquals(self.resTimeSeries.metadata.variable.no_data_value, -9999)
+        self.assertEquals(self.resTimeSeries.metadata.variable.variable_definition, None)
+        self.assertEquals(self.resTimeSeries.metadata.variable.speciation, 'Not Applicable')
+
+        self.assertNotEquals(self.resTimeSeries.metadata.method, None)
+        self.assertEquals(self.resTimeSeries.metadata.method.method_code, 28)
+        method_name = 'Quality Control Level 1 Data Series created from raw QC Level 0 data using ODM Tools.'
+        self.assertEquals(self.resTimeSeries.metadata.method.method_name, method_name)
+        self.assertEquals(self.resTimeSeries.metadata.method.method_type, 'Instrument deployment')
+        method_des = 'Quality Control Level 1 Data Series created from raw QC Level 0 data using ODM Tools.'
+        self.assertEquals(self.resTimeSeries.metadata.method.method_description, method_des)
+        self.assertEquals(self.resTimeSeries.metadata.method.method_link, None)
+
+        self.assertNotEquals(self.resTimeSeries.metadata.processing_level, None)
+        self.assertEquals(self.resTimeSeries.metadata.processing_level.processing_level_code, 1)
+        self.assertEquals(self.resTimeSeries.metadata.processing_level.definition, 'Quality controlled data')
+        explanation = 'Quality controlled data that have passed quality assurance procedures such as ' \
+                      'routine estimation of timing and sensor calibration or visual inspection and removal ' \
+                      'of obvious errors. An example is USGS published streamflow records following parsing ' \
+                      'through USGS quality control procedures.'
+        self.assertEquals(self.resTimeSeries.metadata.processing_level.explanation, explanation)
+
+        self.assertNotEquals(self.resTimeSeries.metadata.time_series_result, None)
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.units_type, 'Temperature')
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.units_name, 'degree celsius')
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.units_abbreviation, 'degC')
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.status, 'Unknown')
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.sample_medium, 'Surface Water')
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.value_count, 1441)
+        self.assertEquals(self.resTimeSeries.metadata.time_series_result.aggregation_statistics, 'Average')
+
+    def test_metadata_extraction_on_content_file_add(self):
```

Comment:
these tests both assert all the same things, just triggered by creation versus add of the file?

Instead, maybe there should be one test for all these assertions that triggers the mechanism consuming this file, and separate tiny test cases that only assert that routine is triggered on creation and addition.

When multiple tests assert the same thing and do so redundantly, it creates a risk they diverge.